components/idp-mgt/org.wso2.carbon.idp.mgt/src/main/java/org/wso2/carbon/idp/mgt/IdentityProviderManager.java

```diff
@@ -3111,4 +3112,43 @@ private Property extractPropertyFromMetadata(MetadataConverter metadataConverter
         }
         return propertyWithName;
     }
+
+    /**
+     * Set the confidential status of federated authenticator properties using metadata.
+     * @param identityProvider Identity Provider.
```

Comment:
Keep a new line after the method description.

Reply:
Fixed with https://github.com/wso2/carbon-identity-framework/pull/4766